uint8 mobility
string[] busyHands
string[] hasObjects
string[] skeletonNames
Joint[] skeletonJoint
Entity meEntity

